pkg/kdump/kdump-view.jsx

```diff
@@ -263,12 +265,43 @@
     }
 
     handleTestSettingsClick() {
+        // if we have multiple targets defined, the config is invalid
+        const target = this.props.kdumpStatus.target;
+        let verifyMessage;
+        if (!target.multipleTargets) {
+            let path = target.path || "/var/crash";
+            if (target.type === "local") {
+                verifyMessage = fmt_to_fragments(
+                    ' ' + _("Results of the crash will be stored in $0 as $1, if kdump is properly configured."),
+                    <span className="ct-monospace">{path}</span>,
+                    <span className="ct-monospace">vmcore</span>);
+            } else if (target.type === "ssh") {
+                verifyMessage = fmt_to_fragments(
+                    ' ' + _("Results of the crash will be copied through SSH to $0 as $1, if kdump is properly configured."),
+                    <span className="ct-monospace">{`${target.server}:${path}`}</span>,
+                    <span className="ct-monospace">vmcore</span>);
+            } else if (target.type === "nfs") {
+                path = path[0] === '/' ? path : '/' + path;
+                verifyMessage = fmt_to_fragments(
+                    ' ' + _("Results of the crash will be copied through NFS at $0 as $1, if kdump is properly configured."),
+                    <span className="ct-monospace">{`${target.server}:${target.export}${path}`}</span>,
+                    <span className="ct-monospace">vmcore</span>);
+            }
+        }
+
         // open a dialog to confirm crashing the kernel to test the settings - then do it
         const dialogProps = {
             title: _("Test kdump settings"),
-            body: (
-                <span>{_("This will test kdump settings by crashing the kernel and thereby the system. Depending on the settings, the system may not automatically reboot and the process may take a while.")}</span>
-            )
+            body: (<TextContent>
+                <Text component={TextVariants.p}>
+                    {_("Test kdump settings by crashing the kernel. This may take a while and the system might not automatically reboot. Do not purposefully crash the system while any important task is running.")}
+                </Text>
+                {verifyMessage && <Text component={TextVariants.p}>
+                    {verifyMessage}
+                </Text>}
+            </TextContent>),
+            showClose: true,
+            titleIconVariant: "warning",
```

Comment:
These 10 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19107-20230817-093415-bc806837-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)